db/migrate/20250429103738_create_api_tokens.rb

```diff
@@ -0,0 +1,13 @@
+class CreateAPITokens < ActiveRecord::Migration[8.0]
+  def change
+    create_table :api_tokens do |t|
```

Comment:
discussed in person: this wasn't part of the spike but we can keep for now among a few extra things to get the API going :)